docker/ui_tests.sh

```diff
@@ -54,6 +54,7 @@ pegasus_port: 3000
 properties_encryption_key: $PROPERTIES_ENCRYPTION_KEY
 saucelabs_authkey: $SAUCE_ACCESS_KEY
 saucelabs_username: $SAUCE_USERNAME
+saucelabs_tunnel_name: build-$CIRCLE_BUILD_NUM
```

Comment:
super-tiny nit; should this be something like `cdo-tunnel-$CIRCLE_BUILD_NUM` (or even `cdo-tunnel-ui-tests-$CIRCLE_BUILD_NUM`), for parallelism with what we have in README.md?

Reply:
I like it. I originally had it as straight `cdo-tunnel` for full parallelism until I realized that having a unique ID per build was actually important. cdo-tunnel-#### makes great sense

Reply:
changed